cfme/tests/openstack/cloud/test_volume_backup.py

```diff
@@ -53,10 +53,11 @@ def incremental_backup(volume_backup, provider):
     volume = volume_backup.appliance.collections.volumes.instantiate(volume_backup.volume, provider)
 
     # create incremental backup for a volume with existing backup
-    backup_name = fauxfactory.gen_alpha()
-    volume.create_backup(backup_name, incremental=True)
-    incremental_backup = backup_collection.instantiate(backup_name, provider)
-    yield incremental_backup
+    if volume.status == 'available':
+        backup_name = fauxfactory.gen_alpha()
+        volume.create_backup(backup_name, incremental=True)
+        incremental_backup = backup_collection.instantiate(backup_name, provider)
+        yield incremental_backup
```

Comment:
In the case where volume.status is not available, this won't yield or return, and L63 will result in an UnboundLocal when `incremental_backup` was never created.

There needs to be an else clause here with a `pytest.skip` call. :)

Reply:
Good change, still seeing a test failure in PRT with:

```
>               raise AssertionError('assert_no_error: {}: {}'.format(message.type, message.text))
E               AssertionError: assert_no_error: error: Unable to create backup for Cloud Volume "PurJWHUPoY": Invalid backup: The parent backup must be available for incremental backup.

/usr/lib/python2.7/site-packages/widgetastic_patternfly/__init__.py:228: AssertionError
AssertionError
assert_no_error: error: Unable to create backup for Cloud Volume "PurJWHUPoY": Invalid backup: The parent backup must be available for incremental backup.
```

Reply:
The point is that locally both tests are passing. what would you suggest me to do further